weaviate/collections/classes/config_vectors.py

```diff
@@ -449,6 +454,40 @@ def text2vec_contextionary(
             vector_index_config=_IndexWrappers.single(vector_index_config, quantizer),
         )
 
+    @staticmethod
+    def text2vec_model2vec(
+        *,
+        name: Optional[str] = None,
+        quantizer: Optional[_QuantizerConfigCreate] = None,
+        source_properties: Optional[List[str]] = None,
+        vector_index_config: Optional[_VectorIndexConfigCreate] = None,
+        api_endpoint: Optional[str] = None,
+        vectorize_collection_name: bool = True,
+    ) -> _VectorConfigCreate:
+        """Create a vector using the `text2vec_model2vec` module.
+
+        See the [documentation](https://weaviate.io/developers/weaviate/modules/retriever-vectorizer-modules/text2vec-model2vec)
+        for detailed usage.
+
+        Args:
+            name: The name of the vector.
+            api_endpoint: The base URL to use where API requests should go. Defaults to `None`, which uses the server-defined default.
+                Docker users may need to specify an alias, such as `http://host.docker.internal:11434` so that the container can access the host machine.
```

Comment:
Does this come from Ollama? This shouldn't be needed w/ docker compose right?

Reply:
Oh, I thought it is a local module that is running in docker - what would be a better text?

Reply:
done!